### DISCUSSION TITLE
Copilot doesnt recognize subscription

### DISCUSSION THREAD

**Discussion by WouterComello** [community/community#31998] [Copilot Conversations] [answered]
After my free trail ended and continued with the paid version, I can no longer login to copilot in VSCode, when pressing sign in it redirects me to my billing page, and doesnt log me in, any solutions for this?

**Comment by davecheney** [comment, marked as answer]
@WouterComello i'm sorry you are having difficulties. 
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by AndreuSCK** [reply]
Hi, there is any chance you could check my issue? https://github.com/community/community/discussions/32073

thanks

**Reply by ajay-dhangar** [reply]
How to use It?
